crates/bevy_utils/src/default.rs

```diff
@@ -0,0 +1,25 @@
+/// Used to make initializing structs with defaults easier:
+/// ```
+/// use bevy_utils::default;
+///
+/// #[derive(Default)]
+/// struct Foo {
+///   bar: usize,
+///   baz: usize,
+/// }
+///
+/// // Normally you would do this:
```

Comment:
IMO this should link to / demonstrate the general application of "struct update syntax". This is a serious stumbling block for new users, and having a term to search demystifies things a lot.